src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -491,6 +529,60 @@ private async Task RegisterHelpersAndPartialsAsync()
             });
         }
 
+        private async Task SendOrganizationInviteEmailAsync(MailMessage message, OrganizationUserInvitedViewModel model)
+        {
+            await AddMessageContentAsync(message, "OrganizationUserInvited", model);
+            message.Category = "OrganizationUserInvited";
+            await _mailDeliveryService.SendEmailAsync(message);
+        }
+
+        private async Task SendBulkAsync<T>(string templateName,
+            IEnumerable<(MailMessage message, T model)> messageModels, Func<MailMessage, T, Task> fallbackSender,
+            params string[] unescapedModelProperties)
+        {
+            if (_mailDeliveryService is AmazonSesMailDeliveryService sesService && messageModels.Skip(1).Any())
```

Comment:
Maybe a better option would be to add the bulk send methods to the interface and then add an additional property that is set to true/false for SupportBulkSending or something like that.

Reply:
I'm ok going that route, but I don't really like the pattern of interfaces having feature toggles like it. To me it makes the interface optional